operating/manage-user-access/configure-ocp-oauth.adoc

```diff
@@ -20,6 +20,7 @@ include::modules/configure-ocp-oauth-identity-provider.adoc[leveloffset=+1]
 ----
 $ oc -n stackrox set env deploy/central ROX_ENABLE_OPENSHIFT_AUTH=true
 ----
+* For access rules, the key "Name" should be used to reference the user name returned from the {ocp} OAuth server.
```

Comment:
Compared to other authentication providers we provide, the openshift auth one doesn't provide a test feature with which you can 
- test login
- review the returned values for i.e. `userid`, `name`, `groups`

Do you think it's worthwhile to have an example here that would showcase what value within the configuration would be mapped to `userid`, `name`, `groups` respectively @gaurav-nelson ?

Reply:
Thank you @dhaus67 yes, I agree having an example would be helpful indeed.

Reply:
I've added some sample values now, let me know what you think.